keyboards/qck75/v1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,585 @@
+			0xfb, 0x7b, 0xfb, 0xfb, 0x07, 0xff, 0xfe, 0x00, 0x00, 
+    	};
+	    static const char PROGMEM scroll2[KB_STAT_SIZE] = {
+	    	0x00, 0x3f, 0x7f, 0x70, 0x6f, 0x6b, 0x6b, 0x6a, 0x68, 
+			0x6a, 0x6b, 0x6b, 0x6f, 0x70, 0x7f, 0x3f, 0x00, 0x00
+	    };
+		oled_set_cursor(SCROLL_X, SCROLL_Y);
+	    oled_write_raw_P(scroll, KB_STAT_SIZE);
+	    oled_set_cursor(SCROLL_X, SCROLL_Y + 1);
+	    oled_write_raw_P(scroll2, KB_STAT_SIZE);
+		}
+	else {
+		static const char PROGMEM scroll[KB_STAT_SIZE] = {
+	        0x00, 0x00, 0x00, 0xf8, 0x04, 0x04, 0x84, 0x04, 0xf4, 
+			0x04, 0x84, 0x04, 0x04, 0xf8, 0x00, 0x00, 0x00, 0x00, 
+    	};
+	    static const char PROGMEM scroll2[KB_STAT_SIZE] = {
+	    	0x00, 0x00, 0x00, 0x0f, 0x10, 0x14, 0x14, 0x15, 0x17, 
+			0x15, 0x14, 0x14, 0x10, 0x0f, 0x00, 0x00, 0x00, 0x00
+	    };
+		oled_set_cursor(SCROLL_X, SCROLL_Y);
+	    oled_write_raw_P(scroll, KB_STAT_SIZE);
+	    oled_set_cursor(SCROLL_X, SCROLL_Y + 1);
+	    oled_write_raw_P(scroll2, KB_STAT_SIZE);
+	}
+}
+
+static void render_winstat(int WIN_X, int WIN_Y)
+{
+	static const char PROGMEM winlogo[WIN_STAT_SIZE] = {
+		0x7C, 0x7C, 0x7C, 0x7E, 0x00, 0x7E,
+	    0x7E, 0x7E, 0x7F, 0x7F, 0x7F, 0x00,
+	};
+	static const char PROGMEM winlogo2[WIN_STAT_SIZE] = {
+	    0x1F, 0x1F, 0x1F, 0x3F, 0x00, 0x3F,
+	    0x3F, 0x3F, 0x7F, 0x7F, 0x7F, 0x00,
+	};
+	
+	oled_set_cursor(WIN_X, WIN_Y);
+	oled_write_raw_P(winlogo, WIN_STAT_SIZE);
+	oled_set_cursor(WIN_X, WIN_Y + 1);
+	oled_write_raw_P(winlogo2, WIN_STAT_SIZE);
+	
+	oled_set_cursor(WIN_X + 2, WIN_Y);
+	oled_write("WINKEY", false);
+	
+	oled_set_cursor(WIN_X + 2, WIN_Y + 1   );
+	if(isguiDisabled)
+    {
+    	oled_write("DISABLED", false);
+	}
+	else {
+		oled_write("ENABLED  ", false);
+	}
+}
+
+bool oled_task_user(void) {
+	current_wpm = get_current_wpm();
+	led_usb_state = host_keyboard_led_state();
+	render_animation(13, 3);
+	oled_set_cursor(0, 0);
+    // Host Keyboard Layer Status
+    switch (get_highest_layer(layer_state)) {
+        case MAIN:
+            oled_write("MAIN  ", false);
+            break;
+        case SECOND:
+            oled_write("SECOND", false);
+            break;
+        case THIRD:
+            oled_write("THIRD ", false);
+            break;
+        default:
+            oled_write("FOURTH", false);
+    }
+	render_caps(0, 2);
+	render_num(3, 2);
+    render_scroll(6, 2);
+	render_winstat(1, 5);   
+	
+    
+    oled_set_cursor(12, 0);
+    oled_write("WPM: ", false);
+    oled_write(get_u8_str(current_wpm, '0'), false);
+    return false;
+}
+
+#endif
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	switch(keycode) {
+		case GU_TOGG:
+			if(record->event.pressed && isguiDisabled) {
+				isguiDisabled = false;
+				break;
+			}
+			else if(record->event.pressed)  {
+				isguiDisabled = true;
+			}
+			break;
+		case KC_LCTL:
+		case KC_RCTL:
+			if(record->event.pressed) {
+				isSneaking = true;
+			} else {
+				isSneaking = false;
```

Comment:
`get_mod() & MOD_MASK_CTRL` could be used instead of this, and would support mod tap and other keycodes.
`(get_mod()|get_weak_mods()|get_oneshot_mods()) & MOD_MASK_CTRL` would be even more complete, too.